packages/angular/.storybook/preview.js

```diff
@@ -0,0 +1,21 @@
+import './polyfills.js';
+
+import { addDecorator, addParameters } from '@storybook/angular';
+import { withKnobs } from '@storybook/addon-knobs';
+import { CarbonG100 } from './theme';
+
+// load global styles
+require('!style-loader!css-loader!postcss-loader!sass-loader!./preview.scss');
+
+addParameters({
+	options: {
+		theme: CarbonG100,
```

Comment:
Are the two packages not going to use the same theme? Should this be G10?

Reply:
Good catch ... We use the G100 theme for the storybook UI in `carbon-components-angular`, but I'll switch it over to G10 to match the React package